examples/api/lib/material/platform_menu_bar/platform_menu_bar.0.dart

```diff
@@ -0,0 +1,116 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for PlatformMenuBar
+import 'package:flutter/material.dart';
+
+void main() => runApp(const SampleApp());
+
+enum MenuSelection {
+  about,
+  showMessage,
+}
+
+class SampleApp extends StatelessWidget {
+  const SampleApp({Key? key}) : super(key: key);
+
+  static const String _title = 'MenuBar Sample';
+
+  @override
+  Widget build(BuildContext context) {
+    return const MaterialApp(
+      title: _title,
+      home: Scaffold(body: MyMenuBarApp()),
+    );
+  }
+}
+
+class MyMenuBarApp extends StatefulWidget {
+  const MyMenuBarApp({Key? key}) : super(key: key);
+
+  @override
+  State<MyMenuBarApp> createState() => _MyMenuBarAppState();
+}
+
+class _MyMenuBarAppState extends State<MyMenuBarApp> {
+  bool get showMessage => _showMessage;
+  bool _showMessage = false;
+  String _message = 'Hello';
+  set showMessage(bool value) {
+    if (_showMessage != value) {
+      setState(() {
+        _showMessage = value;
+      });
+    }
+  }
+
+  void _handleMenuSelection(MenuSelection value) {
+    switch (value) {
+      case MenuSelection.about:
+        showAboutDialog(
+          context: context,
+          applicationName: 'MenuBar Test',
```

Comment:
It's odd that this differs from SampleApp._title, but then I hope we don't need _title at all.